.reuse/dep5

```diff
@@ -223,3 +223,6 @@ License: LGPL-3.0-only
 Files: .prettierignore
 Copyright: 2020 RizinOrg <info@rizin.re>
 License: LGPL-3.0-only
+
+The following files have no copyright and licensing information:
```

Comment:
Please use the proper syntax of this file, see this section above, you could just extend the pattern to match it:
```
Files: test/prj/*.rzdb
Copyright: 2020 RizinOrg <info@rizin.re>
License: LGPL-3.0-only
```